src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -735,12 +744,25 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
     targetWorkspace: string,
     targetWorkspaceName: string
   ) => {
-    const { http, notifications } = this.props;
+    const { notifications } = this.props;
+    const { workspaceClient } = this.state;
```

Comment:
I'd simply call `const workspaceClient = workspace.client$.getValue()` here instead of storing it to a local state